frontend/src/pages/Admin/Posts.tsx

```diff
@@ -33,6 +33,8 @@ export const AdminPostsPage = () => {
     }
   );
 
+  const status = useRole();
```

Comment:
I know that this is just semantics, but we should use role as the variable name, because status could mean something else.